pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
   'cytoolz>=0.12.1',
   'deepdiff>=5.6.0',
   'devtools>=0.6',
+  'diskcache>=5.6.3',
```

Comment:
This is just a reminder for @egparedes to generate the new requirements-dev.txt file once this PR is ready.